pyvista/plotting/axes.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, show_actor=False, actor_scale=1, line_width=1.0, symmetric=Fa
         self.mapper.SetInputConnection(self.GetOutputPort())
         # Add the axes actor
         self.actor = Actor(mapper=self.actor)
```

Comment:
Drive by comment but should this line be?:

```suggestion
        self.actor = Actor(mapper=self.mapper)
```